src/index.html

```diff
@@ -1,15 +1,189 @@
+            id="agree"
+            name="agree"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+
+        <legend>Registration</legend>
+
+        <label for="mail" class="form-field" >
+          E-mail:
+          <input
+            type="email"
+            id="mail"
+            name="mail"
+            placeholder="email@example.com"
+            required>
+        </label>
+
+        <label for="pas">
+          Password:
+          <input
+            type="password"
+            id="pas"
+            name="pas"
+            minlength="8"
+            maxlength="32"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+
+        <legend>An interesting fact about you!</legend>
+
+        <label for="Yes" class="form-field">
+          Do you love cats?
+          <label for="Yes">
+            <input
+              type="radio"
+              id="Yes"
+              name="cats">
+            Yes
+          </label>
+          <label for="No">
+            <input
+              type="radio"
+              id="No"
+              name="cats">
+            No
+          </label>
+        </label>
+
+        <label for="color" class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            id="color"
+            name="color"
+            required>
+        </label>
+
+        <label for="time" class="form-field" >
+          What time do you go to bed?
+          <input
+            type="time"
+            id="time"
+            name="time"
+            required>
+        </label>
+
+        <label for="cars" class="form-field">
+          What are you favorite brand of cars?
+
+          <select
+            name="cars"
+            id="cars"
+            multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label for="rate">
+          How do you rate our work?
+          <input
+            type="range"
+            id="rate"
+            name="rate"
+            min="0"
+            max="10"
+            value="0"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+
+        <legend>Additional info:</legend>
+
+        <label for="Coments" class="form-field" >
+          Coments:
+          <textarea
+            name="Coments"
+            id="Coments"
+            cols="30"
+            rows="2">
+          </textarea>
+        </label>
+
+        <label for="recommend">
+          Would you recommend us?
+          <select name="recommend" id="recommend">
+            <option value="Yes" selected>Yes</option>
+            <option value="No">No</option>
+          </select>
+        </label>
+      </fieldset>
+      <button
+        type="submit"
+        class="field"
+        onsubmit="onSubmit()">
```

Comment:
Fix the indentations and move `onsubmit` attribute to `form` tag.
```suggestion
      <button
        type="submit"
        class="field"
        onsubmit="onSubmit()"
     >
```